khl/client.py

```diff
@@ -148,10 +148,14 @@ def me(self) -> User:
             return self._me
         raise ValueError('not loaded, please call `await fetch_me()` first')
 
-    async def fetch_user(self, user: Union[User, str]) -> User:
+    async def fetch_user(self, user: Union[User, str], channel: Union[Channel, str] = None) -> User:
```

Comment:
should be guild instead of channel?
according to https://developer.kookapp.cn/doc/http/user#%E8%8E%B7%E5%8F%96%E7%9B%AE%E6%A0%87%E7%94%A8%E6%88%B7%E4%BF%A1%E6%81%AF